Dockerfile

```diff
@@ -29,7 +29,22 @@ WORKDIR /usr/local/src
 COPY requirements.txt .
 RUN pip install -r requirements.txt
 
+RUN apt update
+RUN apt -y install tesseract-ocr
+RUN pip install pytesseract
+
+# an empty directory for test output
+RUN mkdir test_txt
+
+# install languages
+RUN apt-get -y install tesseract-ocr-nld tesseract-ocr-spa
+
+RUN pip install langdetect
+RUN pip install pycountry
```

Comment:
I believe these can/should be removed and use `requirements.txt` and a single `apt-get install` instead